ngmix/em_nb.py

```diff
@@ -12,7 +12,7 @@
     gauss2d_set_norm,
     gmix_get_e1e2T,
 )
-from .fastexp import expd
+from .fastexp import exp5, FASTEX_MAX_CHI2
```

Comment:
Did we want to make this `FASTEXP_MAX_CHI2` (notice the `P`)?

Reply:
good catch